build/KoreBuild.sh

```diff
@@ -118,6 +118,15 @@ fi
 
 if [ "$(uname)" == "Darwin" ]; then
     ulimit -n 2048
+
+    # Check that OS is 10.12 or newer
+    osx_version="$(sw_vers | grep ProductVersion | awk '{print $2}')"
```

Comment:
Can't you just grep against `ProductVersion +10\.1[2-9](\.d+)?` - if found you are good otherwise you exit ?